web/po/Makefile

```diff
@@ -10,6 +10,7 @@ MO:=$(patsubst %.po,mo/%/LC_MESSAGES/osmose-frontend.mo,$(PO))
 all: osmose-frontend.pot %.po mo
 
 vue.pot: $(VUEFILES)
+	[ -f ../node_modules/easygettext/src/extract-cli.js ] || (echo 'easygettext missing. Please install node dependencies with `npm install --dev`' && exit 1)
```

Comment:
That's a very good idea, thanks.

Instead of a `[ -f ]`, would it be better to use a Makefile entry? Something like:
```
../node_modules/easygettext/src/extract-cli.js:
    $(error 'easygettext missing. Please install node dependencies with `npm install --dev`')

vue.pot: $(VUEFILES) ../node_modules/easygettext/src/extract-cli.js
```

Reply:
Ohhh neat idea ! Done.